gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://mirrors.cloud.tencent.com/gradle/gradle-7.6-bin.zip
```

Comment:
- For people in other countries, using the tencent‘s source by default will slow down the download speed.
- I prefer to use the default configuration here.

Reply:
ok